src/Aspire.Dashboard/Configuration/PostConfigureDashboardOptions.cs

```diff
@@ -31,20 +31,37 @@ public void PostConfigure(string? name, DashboardOptions options)
         {
             options.Otlp.GrpcEndpointUrl = otlpGrpcUrl;
         }
+        else if (_configuration[DashboardConfigNames.Legacy.DashboardOtlpGrpcUrlName.ConfigKey] is { Length: > 0 } legacyOtlpGrpcUrl)
```

Comment:
Can these use the new IConfigurationExtensions methods that checks both?

Reply:
I don't think we'd want to make falling back on an empty string the default behavior. I worry it would cause some issue somewhere where a string could be empty.

But in this situation, what about adding an optional parameterto the new methods? Should be a minor change and would make the code here cleaner.

Reply:
Yep sounds good.

Reply:
OK done.